apps/openmw/engine.cpp

```diff
@@ -140,6 +138,7 @@ void OMW::Engine::frame(float frametime)
             mEnvironment.getMechanicsManager()->update(frametime,
                 guiActive);
         }
+        mEnvironment.getWorld()->markCellAsUnchanged();
```

Comment:
I used it in actors.cpp.

bool cellChanged = MWBase::Environment::get().getWorld()->hasCellChanged();
 
if (cellChanged)
          player.getClass().getCreatureStats(player).setHitAttemptActorId(-1);

Maybe another way to do it is to just check if the actor is located in a different cell, if there is a function handy for that. I'll take a look at it again later.